packages/vite/src/module-runner/evaluatedModules.ts

```diff
@@ -124,7 +124,13 @@ export class EvaluatedModules {
 }
 
 // unique id that is not available as "$bare_import" like "test"
-const prefixedBuiltins = new Set(['node:test', 'node:sqlite'])
+// https://nodejs.org/api/modules.html#built-in-modules-with-mandatory-node-prefix
+const prefixedBuiltins = new Set([
+  'node:sea',
+  'node:sqlite',
```

Comment:
It's not on the list in the link, but this is also prefix-only (https://nodejs.org/docs/latest/api/sqlite.html#:~:text=This%20module%20is%20only%20available%20under%20the%20node%3A%20scheme.%20The%20following%20will%20not%20work%3A).